graphviz failures on CI

There are some failures popping up in CI. I assume they have to do with a new release of graphviz or something. I'm taking a look and will report back 

```python-traceback
=================================== FAILURES ===================================
________________________ test_visualize_highlevelgraph _________________________
[gw0] linux -- Python 3.8.12 /usr/share/miniconda3/envs/test-environment/bin/python

    @pytest.mark.skipif("not da")
    @pytest.mark.skipif(
        sys.flags.optimize, reason="graphviz exception with Python -OO flag"
    )
    def test_visualize_highlevelgraph():
        graphviz = pytest.importorskip("graphviz")
        with tmpdir() as d:
            x = da.arange(5, chunks=2)
            viz = x.dask.visualize(filename=os.path.join(d, "mydask.png"))
            # check visualization will automatically render in the jupyter notebook
>           assert isinstance(viz, graphviz.dot.Digraph)
E           AttributeError: module 'graphviz.dot' has no attribute 'Digraph'

dask/tests/test_base.py:1021: AttributeError
```

--- a/dask/tests/test_base.py
+++ b/dask/tests/test_base.py
@@ -1018,7 +1018,7 @@ def test_visualize_highlevelgraph():
         x = da.arange(5, chunks=2)
         viz = x.dask.visualize(filename=os.path.join(d, "mydask.png"))
         # check visualization will automatically render in the jupyter notebook
-        assert isinstance(viz, graphviz.dot.Digraph)
+        assert isinstance(viz, graphviz.Digraph)
 
 
 @pytest.mark.skipif(

